src/lcmap/aardvark/espa.clj

```diff
@@ -9,6 +9,83 @@
 
 ;; transformation functions
 
+(def ubid-tags
```

Comment:
This might ultimately end up in the tile-spec ns, but it makes sense here for now.